recipes/icu/all/conandata.yml

```diff
@@ -27,6 +30,8 @@ sources:
     url: "https://github.com/unicode-org/icu/releases/download/release-68-2/icu4c-68_2-src.tgz"
     sha256: "c79193dee3907a2199b8296a93b52c5cb74332c26f3d167269487680d479d625"
 patches:
+  "75go.1":
```

Comment:
```suggestion
  "75.1":
```

Reply:
It happens. I usually blame my code editor, whether or not that's what actually caused it ;)